packages/@expo/cli/package.json

```diff
@@ -113,11 +113,15 @@
     ]
   },
   "peerDependencies": {
+    "@expo/router-server": "*",
```

Comment:
We have to make a decision here. Either:
- take on `@expo/router-server` as a direct dependency (and maintain API compatibility with `expo-router`
- re-export `@expo/router-server` from `expo-router` (?)

The former makes more sense to me, but hard to tell right now. Just noting: peer dependency here is incorrect as users won't install this directly and the optional peer can't pick up on this being depended on by `expo-router`

The former is preferable, since it also forces us to define an exposed API.

tl;dr: we can do `expo-router -> @expo/router-server` or `@expo/router-server -> expo-router` but not both